app/views/events/show.html.erb

```diff
@@ -41,6 +45,13 @@
   <% end %>
 </p>
 
+<p>
+  <strong>Share on Facebook</strong>
+    <%=link_to 'http://www.facebook.com/sharer.php?u='+request.fullpath, :target => :blank do %>
```

Comment:
This didn't work when I tried it?

![image](https://user-images.githubusercontent.com/25331886/56930735-a0f9b680-6a92-11e9-9219-df9dc8fc61dc.png)

https://www.facebook.com/sharer.php?u=/events/2

Reply:
This link won't work until the website has an actual URL